src/Microsoft.Data.Analysis/DataFrame.BinaryOperators.cs

```diff
@@ -662,36 +562,6 @@ public partial class DataFrame
             return df.ReverseModulo(value);
         }
 
-        public static DataFrame operator &(DataFrame df, bool value)
```

Comment:
Why are these being removed?

Reply:
Because it'd only be useful when we have a `DataFrame` composed of only Boolean columns(because `&/|/^ ABoolValue` don't make sense on any other column). I think the feeling in the API review was that it didn't make sense to keep this API for a corner case. The operators are still defined on the base `DataFrameColumn` type though, so a statement like `boolColumn & true` is valid.